src/DXRVanilla/DeusEx/Classes/PersonaScreenInventory.uc

```diff
@@ -228,21 +228,43 @@ function EnableButtons()
             }
 
             if (NanoKeyRing(inv) != None) {
+                //Not a regular inventory item, so don't treat it like anything is currently selected
                 btnDrop.DisableWindow();
                 btnRefusal.DisableWindow();
                 btnEquip.DisableWindow();
                 btnUse.DisableWindow();
             } else {
-                if (WeaponMod(inv) != None || AugmentationUpgradeCannister(inv) != None || DeusExWeapon(inv) != None) {
+                //Handle each button individually
+
+                //"Equip" button text
+                if ((inv == player.inHand ) || (inv == player.inHandPending)) {
+                    btnEquip.SetButtonText(UnequipButtonLabel);
+                } else {
+                    btnEquip.SetButtonText(EquipButtonLabel);
+                }
+
+                //"Equip" button
+                if ((ChargedPickup(inv)!=None && ChargedPickup(inv).bIsActive)){
+                    btnEquip.DisableWindow();
+                }
+
+                //"Use" button
+                if (WeaponMod(inv) != None || AugmentationUpgradeCannister(inv) != None || DeusExWeapon(inv) != None ||
+                    AugmentationCannister(inv) != None ||
+                    (ChargedPickup(inv)!=None && ChargedPickup(inv).bIsActive)) {
+
                     btnUse.DisableWindow();
-                } else if (AugmentationCannister(inv) != None) {
+                }
+
+                //"Refusal" button
+                if (AugmentationCannister(inv) != None ||
+                    (ChargedPickup(inv)!=None && ChargedPickup(inv).bIsActive)) {
                     btnRefusal.DisableWindow();
```

Comment:
any reason why we disable refusal for aug cans? also theoretically we could allow refusal of active chargedpickups and add them to the refusal list without dropping them